packages/react-scripts/template/README.md

```diff
@@ -1546,7 +1546,32 @@ script:
 
 #### CircleCI
 
-Follow [this article](https://medium.com/@knowbody/circleci-and-zeits-now-sh-c9b7eebcd3c1) to set up CircleCI with a Create React App project.
+1. Add a folder `.circleci` to project root. 
+2. Add a `config.yml` inside it.
+
+```
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:6
+    steps:
+      - checkout
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
```

Comment:
Recommend 
```
          key: depens-{{ checksum "package.json" }}-{{ checksum "package-lock.json" }}
```

Reply:
Once I faced a bug caused by this:

- I upgraded package-lock.json for a UI-component, but not the package.json.
- Then I had to update a snapshot of that component
- Then in CI environment, test fails because it used the old node_modules and can't match the snapshot